os/arch/arm/src/tiva/chip/lm3s_flash.h

```diff
@@ -70,11 +70,19 @@
 	defined(CONFIG_ARCH_CHIP_TM4C123GH6ZRB) || defined(CONFIG_ARCH_CHIP_TM4C123GH6PMI) || \
 	defined(CONFIG_ARCH_CHIP_CC3200)
 
+
+#if defined(CONFIG_QEMU_SRAM) || defined (CONFIG_QEMU_SDRAM)
+/* These parts all support a 1KiB erase page size and a total FLASH memory size
+ * of 128MB or 128K pages.
+ */
+#define TIVA_FLASH_NPAGES        (128*1024)
```

Comment:
Isn't possible to make it dynamic. for ex. if FLASH_SIZE = 128 * 1024 * 1024 ,   then 
#define TIVA_FLASH_NPAGES  FLASH_SIZE/1024    . This way we don't have to modify this for different memory configurations. what's your opinion ?

Reply:
In my opionion, It just adds one more macro for total flash size then update the existing npage macro. If flash size changes, we need modify flash size macro instead of npage macro.